docs/src/scs/sbom.md

```diff
@@ -0,0 +1,19 @@
+# Software bill of materials (SBOM)
+Software bill of materials (SBOM) is a formal, machine readable document that provides a list of software components that make up the target software and all its dependencies.
+
+## SBOM usage in Ghaf
+Ghaf framework will use SBOMs for:
+- Vulnerability identification: automatic correlation of SBOM against known vulnerabilities
+- Vulnerability remediation: automatic process to suggest fixes for identified vulnerabilities
+- Dependency analysis: categorization of open-source and closed source software dependencies
+- Dependency analysis: creation of a directed acyclic graph
+- Market signal: publish SBOM together with other release artifacts
+
+## SBOM tooling in Ghaf
+Ghaf is based on Nix, therefore, the selected SBOM tooling needs to support creating SBOMs for nix artifacts. As part of the Ghaf project, we have created the sbomnix tool to support SBOM generation for Ghaf and, more generally, for any nix targets. For more details on the SBOM tooling, see [sbomnix](https://github.com/tiiuae/sbomnix#sbomnix) and [nixgraph](https://github.com/tiiuae/sbomnix/blob/main/doc/nixgraph.md#nixgraph).
+
```

Comment:
It would be great to add information on the SPDX and CycloneDX, state the differences and perhaps say something like "we have chosen CycloneDX, because ...". Some information on the fields that we are using can be added). Perhaps open up on which fields are for what.

Reply:
Thanks, I'll add some more information around the SBOM formats in a moment.

Reply:
Please see https://github.com/tiiuae/ghaf/commit/fe269404240597ce1b0bf4d54e553295669c921a